capstone/src/main/webapp/group.html

```diff
@@ -1,101 +1,94 @@
-        <div class="mark-completed">
-          <span class="mark-completed-text">Mark Completed:</span>
-          <span class="check-container">
-            <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike">
-            <label for="vehicle1"></label>
+				<input id='fileid' type='file' name='image' />
+
+				<button class="post-btn align-vertical" type="image" form="post-form"><img class="medium-icon" src="images/send_plane.png" alt /></button>
+			</form>
+		</div>
+
+		<div class="challenge-container">
+			<h2>
+				Challenge of the Week
+			</h2>
+			<p id="weekly-challenge"></p>
+			<p id="due-date"></p>
+			<div class="mark-completed">
+				<span id="mark-completed-text">Mark Completed:</span>
+				<span class="check-container">
+            <input type="checkbox" class="challenge-checkbox" onchange="markChallenge(this.id, this.checked)">
+            <label id="challenge-label"></label>
           </span>
-        </div>
-      </div>
-          
-      <div class="poll-container">
-        <div class="inner-poll-container">
-          <h2>
-            Poll
-          </h2>
-        <div id="options-container"></div>
-      </div>
-      
-      <input id="input-box" name="input-box" placeholder="Add a suggestion" onkeydown="if (event.keyCode == 13) addPollOption()"></input>
-          
-    </div>
-        
-      <div class="post-feed">
-          <div id="posts-container"></div>
-      </div>
-    </div>
-    
-    <template id="option-template">
-      <div class="option-container">
-        <p></p>
-        <svg width="690" height="55">
-          <rect height="55" width="690" class="poll-rect" />
-          <rect height="55" id="inner-bar" class="poll-inner-rect" />
-        </svg>
+			</div>
+		</div>
+
+		<div class="poll-container">
+			<div class="inner-poll-container">
+				<h2>
+					Poll
+				</h2>
+				<div id="options-container"></div>
+			</div>
 
-        <span class="check-container">
+			<input id="input-box" name="input-box" placeholder="Add a suggestion" onkeydown="if (event.keyCode == 13) addPollOption()"></input>
+
+		</div>
+
+		<div class="post-feed">
+			<div id="posts-container"></div>
+		</div>
+	</div>
+
+	<template id="option-template">
+		<div class="option-container">
+			<p></p>
+			<svg width="690" height="55">
+				<rect height="55" width="690" class="poll-rect" />
+				<rect height="55" id="inner-bar" class="poll-inner-rect" />
+			</svg>
+
+			<span class="check-container">
           <input type="checkbox" onchange="handleCheckboxCount(this.id, this.checked)">
           <label></label>
         </span>
-        <div id="num-votes" class="votes-label"></div>
-      </div>
-    </template>
-    
-    <script>
-      window.onload = getPollOptions();
-    </script>
-  </body>
-</html>
-  
-
-
+			<div id="num-votes" class="votes-label"></div>
+		</div>
+	</template>
 
+	<script>
+		window.onload = getPollOptions();
+	</script>
+</body>
 
-  
+</html>
```

Comment:
I think this is largely my fault for not having noticed the preference in the cloud shell, but I'd recommend changing the tabSize from 4 to 2 (File -> Settings -> Open Preferences -> editor -> tabSize). There looks to be a lot of unnecessary formatting change here, I think mostly coming from that

Reply:
hmm the tab size was already set to 2 i don't know why that formatting happaned